src/components/static/Title.jsx

```diff
@@ -18,13 +22,16 @@ const Title = () => {
           </div>
           <div className="text-lg lg:text-xl font-paragraph text-bear-teal-100">
             {CONFIG.date.toLocaleDateString(undefined, {
-              year: "numeric",
               month: "long",
               day: "numeric",
             })}
+            {"-"}
+            {CONFIG.lastDate.toLocaleDateString(undefined, { day: "numeric" })}
```

Comment:
if u just need the day and year there r also getDay and getFullYear functions

Reply:
o u right